hazelcast/src/main/java/com/hazelcast/cache/impl/CacheDataSerializerHook.java

```diff
@@ -324,6 +327,16 @@ public IdentifiedDataSerializable createNew(Integer arg) {
                 return new CacheBatchInvalidationMessage();
             }
         };
+        constructors[ENTRY_ITERATOR] = new ConstructorFunction<Integer, IdentifiedDataSerializable>() {
+            public IdentifiedDataSerializable createNew(Integer arg) {
+                return new CacheEntryIteratorOperation();
+            }
+        };
+        constructors[ENTRY_ITERATION_RESULT] = new ConstructorFunction<Integer, IdentifiedDataSerializable>() {
+            public IdentifiedDataSerializable createNew(Integer arg) {
+                return new CacheEntryIteratorResult();
```

Comment:
Names can be consistent `iterator` or `iteration`:  `ENTRY_ITERATION_RESULT` vs `CacheEntryIteratorResult `?